elastic/kibana
# [Fleet] Fix get file handler for bundled package

## Summary

Resolve https://github.com/elastic/kibana/issues/166045
That PR allow to retrieve files for bundled package that are not installed, this allow to display the APM integration on premise 


#### Before that change
<img width="600" alt="Screenshot 2023-11-29 at 11 08 12 AM" src="https://github.com/elastic/kibana/assets/1336873/eef73998-566c-4676-93a6-267c736e0691">

#### After

<img width="600" alt="Screenshot 2023-11-29 at 11 06 08 AM" src="https://github.com/elastic/kibana/assets/1336873/77ee3625-87c4-49bb-b608-2f7f28d1b63a">

## Manual tests instructions

* retrieve 8.11.1 bundled packages 
```
cd /tmp
mkdir test-8.11
cd test-8.11
wget https://artifacts.elastic.co/downloads/kibana/kibana-8.11.1-darwin-x86_64.tar.gz
tar -xvf kibana-8.11.1-darwin-x86_64.tar.gz
```

* Configure kibana to use those bundled packages
`xpack.fleet.developer.bundledPackageLocation: /tmp/test-8.11/kibana-8.11.1/node_modules/@kbn/fleet-plugin/target/bundled_packages`

* Navigate to apm and check the integration is displayed http://localhost:5601/app/integrations/detail/apm/overview

## Todo
